app/main/views/services.py

```diff
@@ -503,6 +503,62 @@ def update_section_submission(framework_slug, lot_slug, service_id, section_id,
                                 service_id=service_id))
 
 
+@main.route('/frameworks/<framework_slug>/submissions/<lot_slug>/<service_id>/remove/<question_slug>',
```

Comment:
Good point.  I did try to think of a good reason to keep the longer version of the URL - this is it.